pilot/pkg/xds/ads.go

```diff
@@ -925,14 +925,13 @@ func (s *DiscoveryServer) removeCon(conID string) {
 	}
 }
 
-// Send with timeout if configured.
 func (conn *Connection) send(res *discovery.DiscoveryResponse) error {
-	sendHandler := func() error {
+	sendResponse := func() error {
 		start := time.Now()
 		defer func() { recordSendTime(time.Since(start)) }()
 		return conn.stream.Send(res)
 	}
-	err := istiogrpc.Send(conn.stream.Context(), sendHandler)
+	err := sendResponse()
```

Comment:
I'm curious why this change can solve the issue mentioned in the PR description. When `PILOT_XDS_SEND_TIMEOUT==0`, it seems that the behavior has not changed.

Reply:
PILOT_XDS_SEND_TIMEOUT is set to `20s` when that issue happened. It was a legacy value and later we changed to `0s`

Reply:
Got it, Thanks.